connector/src/it/scala/com/datastax/spark/connector/SparkCassandraITFlatSpecBase.scala

```diff
@@ -170,8 +199,9 @@ trait SparkCassandraITSpecBase
        |""".stripMargin
 
   def createKeyspace(session: CqlSession, name: String = ks): Unit = {
-    session.execute(s"DROP KEYSPACE IF EXISTS $name")
-    session.execute(keyspaceCql(name))
+    val ks_ex = getExecutor(session)
+    ks_ex.execute(newInstance(s"DROP KEYSPACE IF EXISTS $name"))
```

Comment:
Nit: the newInstance() call here caught me off guard.  It makes sense given the import but "newInstance" is pretty generic.  Maybe this should be aliased to "newSimpleStatement" or something?

Not a big deal at all, just something I noticed.

Reply:
I think i'm gonna leave that here since this is test code and ideas should make it clear pretty fast. It does make me sad that I can't just pass strings through, but such is life